.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+How to write a good issue ? Read [this](https://github.com/containous/traefik/blob/master/.github/how-to-write-a-good-issue.md) and the [contributing guide](https://github.com/containous/traefik/blob/master/.github/CONTRIBUTING.md).
```

Comment:
This links are viewed by user who open an issue as text.
If we use relative links, the user can be lost.